lib/datadog/core/configuration.rb

```diff
@@ -236,7 +236,7 @@ def safely_synchronize
           rescue ThreadError => e
             logger_without_components.error(
               'Detected deadlock during datadog initialization. ' \
-              'Please report this at https://github.com/DataDog/dd-trace-rb/blob/master/CONTRIBUTING.md#found-a-bug' \
+              'Please report this at https://github.com/datadog/dd-trace-rb/blob/master/CONTRIBUTING.md#found-a-bug' \
```

Comment:
Nit: Intended? I mean it works but the official casing if the previous.

Reply:
I usually try to lower-case whenever I see it linked like this, trying to convince the world it's not DataDog ;)